### DISCUSSION TITLE
Where is getUpdates method?

### DISCUSSION THREAD

**Discussion by xaosaki** [nksmnf/nestjs-telegram#330] [Q&A]
Hello, i can't find `getUpdates` method? Do you forget add it or you have some reasons to not implement it. Seems it is very common and helpful method.

**Comment by evilsprut** [comment]
Hey, @xaosaki maybe [my module](https://github.com/evilsprut/nestjs-telegraf) will suit you? Sorry for the ads

**Comment by xaosaki** [comment]
@evilsprut , thank you, i saw you module, but i want use maximal simple solution for my purposes

**Comment by evilsprut** [comment]
@xaosaki 🙌

**Comment by jmcdo29** [comment]
Oh wow, okay. Obviously I haven't touched this module since, like, two years ago when I created it over a holiday break to learn about dynamic modules. If @bukhalo's module is being currently maintained, I would highly suggest using it instead. I can try to look into adding more methods, but again, I haven't touched this code in a good while. :sweat_smile: If anyone wants to take over ownership as well, I'd be happy to transfer it :smiley:

**Comment by evilsprut** [comment]
@jmcdo29 hey. I could be the maintainer of this library. You can pass the repository to me or just give me the rights to the repository. `nestjs-telegraf` is not an alternative to this module because someone might want a simple client for the telegram bot api rather than a huge combine like I do.

**Comment by jmcdo29** [comment]
> @jmcdo29 hey. I could be the maintainer of this library. You can pass the repository to me or just give me the rights to the repository. `nestjs-telegraf` is not an alternative to this module because someone might want a simple client for the telegram bot api rather than a huge combine like I do.  

Absolutely you can! If you're on discord we can swap info there: PerfectOrhan31#6003

**Comment by evilsprut** [comment]
Hey, @jmcdo29! I tried to write to you a long time ago, but it seems not the right nickname. I sent a request to PerfectOr**ph**an31#6003 but I'm not sure if it is you. Could you check it out? I can not contact you. Or write me on a@bukhalo.com

**Reply by jmcdo29** [reply]
That is me, but I've generally ignored direct messages on discord that don't have context. I get too many requests otherwise from unknowns. What's up?

**Comment by evilsprut** [comment]
> Hello, i can't find `getUpdates` method? Do you forget add it or you have some reasons to not implement it. Seems it is very common and helpful method.

To be done soon in the next version

**Comment by evilsprut** [comment]
Basic implementation of `getUpdates` method added in `v1.2.0`
